core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java

```diff
@@ -396,8 +395,7 @@ public double get() {
      * @param target    the target position in ticks
      */
     public void setTargetPosition(int target) {
-        setTargetDistance((double)target / (encoder.dpp));
-        targetIsDistance = false;
+        setTargetDistance(target * encoder.dpp);
```

Comment:
Sorry, I'm slow but why would you multiply rather than divide target and dpp?

Reply:
Distance per pulse is distance / tick. So when we set a target position in ticks, to get the distance we do ticks * distance / tick which returns distance.

Reply:
Ahhhh, for some reason I was thinking of it in reverse. Thanks, the PR looks good then.